enterprise/server/invocation_stat_service/invocation_stat_service.go

```diff
@@ -10,10 +10,10 @@ import (
 	"strings"
 	"time"
 
-	"github.com/buildbuddy-io/buildbuddy/enterprise/server/invocation_stat_service/config"
 	"github.com/buildbuddy-io/buildbuddy/server/build_event_protocol/invocation_format"
 	"github.com/buildbuddy-io/buildbuddy/server/environment"
 	"github.com/buildbuddy-io/buildbuddy/server/interfaces"
+	"github.com/buildbuddy-io/buildbuddy/server/static"
```

Comment:
invocation_stat_service shouldn't depend on static, feels like a recipe for future circular dependencies.

Can we just move inovcation_stat_service/config from enterprise to server like we did here: https://github.com/buildbuddy-io/buildbuddy/pull/5245

Reply:
I tried that first. But then I realized that we have several Trends/Heatmap flags that are scattered in different places and `static` is a convenience place to group them up.

Let me move the `trends_heatmap_enabled` flag back out again